rpc/flipt/flipt.yaml

```
@@ -323,3 +323,9 @@ http:
   # method: evaluation count
   - selector: flipt.analytics.AnalyticsService.GetFlagEvaluationsCount
     get: /internal/v1/analytics/namespaces/{namespace_key}/flags/{flag_key}
+
+  # Open Feature Remote Evaluation Protocol (OFREP)
+  #
+  # method: provider configuration
+  - selector: flipt.ofrep.OFREPService.GetProviderConfiguration
+    get: /ofrep/*/ofrep/v1/configuration
```

```suggestion
    get: /ofrep/{namespace_key}/ofrep/v1/configuration
```

Our SDK client generator doesn't really know how to handle this wildcard. So at the moment, it is generating a request to this path as you have it here with an explicit `*` in it. If we made this explicitly the namespace key and added that field to the `GetProviderConfigurationRequest` (even though we're not going to use it), that will ensure the client gets generated in such a way that it doesn't render a path with `*` in it.

Or maybe it doesn't really matter as maybe no one will use the OFREP section of our Go SDK.
We could actually use our magic comment that stops it from getting generated altogether.